docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -691,6 +693,78 @@ System.out.println(description.writeQueries().size() + " queries write to this s
 System.out.println(description.readQueries().size() + " queries read from this stream/table.");
 ``` 
 
+Get metadata about the ksqlDB cluster
```

Comment:
We should update the table of contents at the top of this file with the new sections so they're more discoverable.